src/shadowbox/index.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright 2020 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {DataUsageByUser} from './model/metrics';
```

Comment:
Can't the client depend on those modules directly instead of creating this file?

Notice that what you are exposing here is a function of what the client is consuming. that's kind of a circular dependency.

Reply:
>Notice that what you are exposing here is a function of what the client is consuming. that's kind of a circular dependency.

I have renamed `DataUsageByUser`, which is a misnomer since there is no user model, to `DataUsageByAccessKey`.  Added a TODO to rename `bytesTransferredByUserId` in a backwards compatible way.